### ISSUE TITLE
Archive docs links point to current version pages

### ISSUE THREAD

**Issue by dtrudg** [singularityware/singularityware.github.io#144]
If I am looking at archive docs for 2.3, e.g:

http://singularity.lbl.gov/archive/docs/v2-3/user-guide

Then the links on that page point at the current version (2.4) docs, e.g. the Examples->run link on the 2.3 docs is pointing to....

http://singularity.lbl.gov/docs-run

This just made my head hurt a bit, but will catch out users of older versions if things have changes 2.3 -> 2.4

**Comment by vsoch** [comment]
yes that's a good point, unfortunately it means hard coded paths (e.g., `<a href="/docs-run">Run</a>`. I think we can do work to fix this in future versions, but probably it would be very tenuous to go back and try to manually fix.
